test/unit/lib/censorBirthV0.js

```diff
@@ -0,0 +1,275 @@
+      name: {
+        givenName: "MOTHER FORENAME",
+        surname: "MOTHER SURNAME",
+        fullName: "MOTHER FORENAME MOTHER SURNAME",
+        qualifier: "generated from prepending forename to surname"
+      },
+      birthplace: "Exeter",
+      occupation: "Teacher",
+      maidenSurname: "MAIDEN SURNAME",
+      marriageSurname: "MARRIAGE SURNAME",
+      usualAddress: "ADDRESS 1"
+    },
+    informant: {
+      name: {
+        givenName: "Informant Forename1",
+        surname: "Informant1 Surname",
+        fullName: "Informant Forename1 Informant1 Surname",
+        qualifier: "generated from prepending forename to surname"
+      },
+      usualAddress: "INFORMANT ADDRESS",
+      qualification: "INFORMANT QUALIFICATION",
+      signature: "INFORMANT SIGNATURE"
+    }
+  },
+  systemNumber: 12345,
+  id: 98765,
+  date: "2020-09-23",
+  registrarSignature: "Registrar signature",
+  status: {
+    potentiallyFictitiousBirth: false,
+    correction: "None",
+    cancelled: false,
+    blockedRegistration: false,
+    marginalNote: "Marginal Note",
+    reRegistered: false
+  },
+  previousRegistration: {
+    date: "2020-05-23",
+    systemNumber: 3455667
+  }
+}
+
+const redactedBirthData = {
+  location: {
+    name: "Registration District, Subdistrict town, Adminshire",
+    administrativeArea: "Adminshire",
+    subDistrict: "Subdistrict town",
+    nameQualifier: "generated by concatenating registrationDistrict, subDistrict and administrativeArea",
+    registrationDistrict: "Registration District"
+  },
+  subjects: {
+    child: {
+      originalName: {
+        givenName: null,
+        surname: null,
+        fullName: null,
+        qualifier: null
+      },
+      name: {
+        givenName: "Expensive",
+        surname: "Mistake",
+        fullName: "Expensive Mistake",
+        qualifier: "generated from prepending forename to surname"
+      },
+      dateOfBirth: "2020-09-23",
+      sex: "Male",
+      birthplace: "Bath"
+    },
+    father: {
+      name: {
+        givenName: "FATHER FORENAME",
+        surname: "FATHER SURNAME",
+        fullName: "FATHER FORENAME FATHER SURNAME",
+        qualifier: "generated from prepending forename to surname"
+      },
+      birthplace: "BRISTOL",
+      occupation: null
+    },
+    mother: {
+      name: {
+        givenName: "MOTHER FORENAME",
+        surname: "MOTHER SURNAME",
+        fullName: "MOTHER FORENAME MOTHER SURNAME",
+        qualifier: "generated from prepending forename to surname"
+      },
+      birthplace: "Exeter",
+      occupation: null,
+      maidenSurname: "MAIDEN SURNAME",
+      marriageSurname: "MARRIAGE SURNAME",
+      usualAddress: null
+    },
+    informant: {
+      name: {
+        givenName: null,
+        surname: null,
+        fullName: null,
+        qualifier: null
+      },
+      usualAddress: null,
+      qualification: "INFORMANT QUALIFICATION",
+      signature: null
+    }
+  },
+  systemNumber: 12345,
+  id: 98765,
```

Comment:
I think this could probably be redacted.

Reply:
System number is used in the view:

https://github.com/UKHomeOffice/lev-web/blob/master/views/pages/details.html#L47

And in the Wrangler ID is set from system number (or vice versa)...so not sure why there is both of them but as they have the same source, I've redacted neither

Reply:
I think you are right.
I think they used to be different things but I decided that it was wrong to expose id so I must have just piped system number in to it.